lib/vssh/libssh2.c

```diff
@@ -3281,6 +3281,14 @@ static CURLcode ssh_connect(struct Curl_easy *data, bool *done)
     return CURLE_FAILED_INIT;
   }
 
+  /* Set the packet read timeout if the libssh2 version supports it */
+#ifdef HAVE_LIBSSH2_VERSION
+ if(libssh2_version(0x010B00)) {
+   libssh2_session_set_read_timeout(sshc->ssh_session,
```

Comment:
`libssh2_session_set_read_timeout` should be:
`vssh/libssh2.c: note: did you mean 'libssh2_session_set_timeout'?`

Reply:
See my comment a few minutes ago about not having the build tested against current libssh2 (this patch is for an as-yet unreleased feature of libssh2).  I've now pushed a fix for this mis-build